dracut/30ignition/module-setup.sh

```diff
@@ -33,7 +33,29 @@ install() {
         sgdisk \
         useradd \
         usermod \
-        wipefs
+        wipefs \
+        cryptsetup \
+        clevis \
+        clevis-decrypt \
+        clevis-decrypt-sss \
+        clevis-decrypt-tang \
+        clevis-decrypt-tpm2 \
+        clevis-encrypt-sss \
+        clevis-encrypt-tang \
+        clevis-encrypt-tpm2 \
+        clevis-luks-bind \
+        clevis-luks-common-functions \
+        clevis-luks-unlock \
+        pwmake \
+        tpm2_create \
+        tpm2_createprimary \
+        tpm2_load \
+        tpm2_pcrread \
+        tpm2_pcrlist \
+        tpm2_unseal \
+        jose
+
+    inst_libdir_file 'libtss2-tcti-device.so*'
```

Comment:
Already included by clevis dracut modules.

Reply:
Hmm, since we're adding the clevis dracut modules in a different repo (f-c-c) would it make sense to just leave a comment (like we did for the `inst_multiple` binaries) rather than dropping it?

Reply:
I think I'd prefer removing them from here because we don't actually call out to all these binaries, right? Basically we just use `cryptsetup` and `clevis` directly. Everything else is a dep of clevis. Let's let the clevis dracut modules track those.

The ones needed for encrypting break that a bit, so we do have to list those out explicitly. (Ideally there'd also be a `clevis-encrypt` dracut module upstream that tracks those, though understandably the fact that we're creating and binding LUKS devices from the initrd is pretty special.)